uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/staff-leave-report.js

```diff
@@ -404,6 +404,7 @@ define([
         })
         .then(function (results) {
           vm.sections.expired.data = results.expiredBalanceChangesFlatten.concat(results.expiredTOILS);
+          vm.sections.expired.data = _.sortBy(vm.sections.expired.data, 'expiry_date');
```

Comment:
Is `expiry_date` property a string? If yes, then the sort is performed by comparing strings. Please make sure that `expiry_date` property can be used for sorting by string, otherwise you might want to use moment.js to convert it to timestamp first.